src/Methods.java

```diff
@@ -0,0 +1,132 @@
+public class Methods {
+    public static float countAllSalary(Employee[] employees, float salary) { //сумма затрат на зарплаты
+        float allSalary = 0;
+        for (Employee employee : employees) {
+            allSalary = allSalary + employee.getSalary();
+        }
+        return allSalary;
+    }
+
+    public static Employee minimumSalary(Employee[] employees) { //минимальная зарплата
+        float min = 100000;
+        Employee employee = new Employee("", 0, 0);
+        for (Employee value : employees) {
+            if (value.getSalary() < min) {
+                min = value.getSalary();
+                employee = value;
+            }
+        }
+        return employee;
+    }
+
+    public static Employee maximumSalary(Employee[] employees) { //максимальная зарплата
+        float max = 50000;
+        Employee employee = new Employee("", 0, 0);
+        for (Employee value : employees) {
+            if (value.getSalary() > max) {
+                max = value.getSalary();
+                employee = value;
+            }
+        }
+        return employee;
+    }
+
+    public static float averageSalary(Employee[] employees) { //средняя зарплата
+        float average = 0;
+        for (Employee employee : employees) {
+            average = countAllSalary(employees, employee.getSalary());
+        }
+        average = average / employees.length + average % employees.length;
```

Comment:
```suggestion
        average = average / employees.length;
```
деление и так будет с остатком, а не нацело, поэтому вторая часть не нужна, она неправильный ответ даёт